tables_cleaner/management/commands/clean_tables.py

```diff
@@ -89,10 +89,27 @@ def handle(self, *args, **options):
             if transaction.get_autocommit(self.using):
                 connections[self.using].close()
 
-        if not self.dry_run and self.vacuum:
+        if self.vacuum:
+            self.vacuum_db(connections[self.using])
 
-            self.logger.info('Executing VACUUM FULL (untested !!!)')
-            connection = connections[self.using]
+        self.logger.info("*** clean_tables done.")
+
+    def vacuum_db(self, connection):
+        engine_module = connection.__class__.__module__
+        if 'sqlite' in engine_module or 'spatialite' in engine_module:
+            self.logger.info('Executing Sqlite3 VACUUM')
+
+            if self.dry_run:
+                return
+
+            cursor = connection.cursor()
+            cursor.execute('VACUUM')
+            cursor.close()
+        elif 'postgresql' in engine_name or 'postgis' in engine_name:
```

Comment:
elif 'postgresql' in engine_module or 'postgis' in engine_module

Reply:
Thanks for spotting this!